### DISCUSSION TITLE
Is there method to simulate a low-image-quality scanner to generate image?

### DISCUSSION THREAD

**Discussion by nissansz** [python-pillow/Pillow#8264] [Q&A] [answered]
Is there method to simulate a low-image-quality scanner to generate image?

![image](https://github.com/user-attachments/assets/659f578b-5a4e-477a-aa52-b625b6b9eb28)

**Comment by radarhere** [comment, marked as answer]
That's a very specific request, so Pillow doesn't have a direct feature for it, no.

It seems similar to [a request that you've made before](https://github.com/python-pillow/Pillow/issues/7291#issuecomment-2021495044) - take a look at https://github.com/python-pillow/Pillow/issues/7291#issuecomment-2021519574

**Comment by nissansz** [comment]
Thank you. I guess, you must have many methods to simulate such effects.
